server/utils/placementRequests/statusTag.ts

```diff
@@ -0,0 +1,33 @@
+import { RequestForPlacementStatus } from '../../@types/shared'
+import { StatusTag, StatusTagOptions } from '../statusTag'
+
+export class RequestForPlacementStatusTag extends StatusTag<RequestForPlacementStatus> {
+  static readonly statuses: Record<RequestForPlacementStatus, string> = {
+    awaiting_match: 'Awaiting match',
+    person_arrived: 'Person arrived',
+    person_departed: 'Person departed',
+    person_not_arrived: 'Person not arrived',
+    placement_booked: 'Placement booked',
+    request_rejected: 'Request rejected',
+    request_submitted: 'Request submitted',
+    request_withdrawn: 'Request withdrawn',
+  }
+
+  static readonly colours: Record<RequestForPlacementStatus, string> = {
+    awaiting_match: 'yellow',
+    person_arrived: 'blue',
+    person_departed: 'blue',
+    person_not_arrived: 'blue',
+    placement_booked: 'grey',
+    request_rejected: 'red',
+    request_submitted: 'yellow',
+    request_withdrawn: 'red',
+  }
```

Comment:
Nice! These configuration lists will be easy to maintain.

I think we have the wrong colour for `person_not_arrived`. This status means "the person failed to arrive", so I think red is probably the right colour? 

Does it still make sense for `placement_booked` to be in grey or is blue more appropriate as in Adam's mockup?

Reply:
To be honest I was guessing these colours so happy to defer to these suggestions